examples/error-demo/src/components/ErrorGenerator.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
```

Comment:
I would prefer using standard `.js`, I don't see much value in `.jsx` extension as JSX is handled just fine in regular `.js` files by all the tools.